src/store/options.js

```diff
@@ -11,23 +11,20 @@
 
 import { store } from 'hybrids';
 
-const UPDATE_ACTION_NAME = 'updateOptions';
-
-export function isUpdateOptionsMessage(message) {
-  return message?.action === UPDATE_ACTION_NAME;
-}
-
 export const DNR_RULES_LIST = chrome.runtime
   .getManifest()
   .declarative_net_request.rule_resources.map((r) => r.id);
 
+const UPDATE_OPTIONS_ACTION_NAME = 'updateOptions';
+
 const Options = {
   dnrRules: DNR_RULES_LIST.reduce(
-    (all, rule) => ({ ...all, [rule]: true }),
```

Comment:
this is new logic related to the onboarding, right? should it be in this PR?

Reply:
No, it isn't. It was here all the time.